prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -62,26 +68,53 @@ class LoginCommand : OktaCommand(
         .choice("local", "test", "staging", "prod")
         .default("local", "local")
 
+    private val manualLogin by option(
+        "--manual",
+        help = "Enable to open the login page and sign manually to Okta"
+    ).flag(default = false)
+
+    private val forceRefreshToken by option(
+        "--force",
+        help = "Enable to do a login regardless of any current access tokens"
+    ).flag(default = false)
+
     override fun run() {
         val oktaApp = Environment.get(env).oktaApp ?: abort("No need to login in this environment")
-        val accessTokenFile = readAccessTokenFile()
-        if (accessTokenFile != null && isValidToken(oktaApp, accessTokenFile)) {
+
+        // Load environment variables for the Okta Api
+        // The prefix has to be added as this value is already set for other uses
+        oktaBaseUrl = oktaBaseUrls[oktaApp] ?: error("Invalid app - Okta url")
+        oktaClientId = clientIds[oktaApp] ?: error("Invalid app")
+
+        oktaAuthKey = System.getenv("OKTA_authKey")
```

Comment:
This would seem to break the current behavior?

Reply:
Also, we should figure out how to get around using `System.getenv` and use azure to get the key because this env var cannot be stored in terraform since it is sensitive info?

Reply:
I have moved the env loading to only happen when the key is required, so by default it's ignored

Reply:
My understanding after discussing with DevOps is that terraform is able to be set to pull from the Azure KeyVault